src/llmcompressor/transformers/finetune/data/base.py

```diff
@@ -53,11 +53,7 @@ def __init__(
         self.tokenizer = getattr(self.processor, "tokenizer", self.processor)
 
         if self.tokenizer is not None:
-            # fill in pad token
```

Comment:
Do we not use this no more?

Reply:
Ok you moved it to 
 if hasattr(tokenizer, "pad"):
            collate_fn = DataCollatorWithPadding(tokenizer)

Reply:
No, I moved this logic to `configure_processor`